MANIFEST.in

```diff
@@ -1,14 +1,17 @@
-include AUTHORS.rst
-include LICENSE
-include README.rst
-include requirements.txt
-
-recursive-exclude * __pycache__
-recursive-exclude * *.py[co]
+graft src
+graft tests
+graft requirements
 
+include AUTHORS.rst LICENSE*.rst README.rst
+include regolith/version.py
 recursive-include docs *.rst conf.py Makefile make.bat
```

Comment:
I copied this over from the old version. Do we keep recursive-include or global-include here?

Reply:
Hi Tina! for for MANIFEST.in, we've decided not to include docs in our sdist 

could you please remove:

```
recursive-include docs *.rst conf.py Makefile make.bat
```

Ref: https://github.com/Billingegroup/cookiecutter?tab=readme-ov-file#why-we-decided-to-include-test-files-in-pypi-release

Reply:
But per Tina's question, go with the new graft syntax

Reply:
done!